src/css/jwplayer/flags/floatingplayer.less

```diff
@@ -17,19 +17,19 @@
         margin: 0 auto;
 
         @media screen and (max-width : 480px) {
-            width: 100% !important; /* stylelint-disable-line declaration-no-important */
```

Comment:
we should really do an `important` audit _especially_ being an embedded solution this could be real bad in some cases

Reply:
We most commonly use `important` to override inline styles added by JS. Good news here is we're not using it anymore.